grid-demo/src/app/angular/row-dom-performance/row-dom-performance.component.html

```diff
@@ -1,3 +1,30 @@
+<clr-accordion>
+  <clr-accordion-panel [clrAccordionPanelOpen]="true">
+    <clr-accordion-title>Demo Explanation</clr-accordion-title>
+    <clr-accordion-content>
+      <p>
+        This demo improves performance when there are many rows on a page or when not using pagination. By default, the
+        datagrid would always render every row in the DOM, even rows that were out of view to the user. In this demo,
+        only rows that are visible to the user will be renedred (as well as a buffer of rows above and below this
+        visible space).
+      </p>
+
+      <p>
```

Comment:
Again, an Accessibility comment is needed here. Probably something like "should be used in conjunction with pagination so keyboard or screenreader users can fallback to keyboard navigation using the paginator"